force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -12098,9 +12106,25 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <categories>Payment</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Refund rror message when Elevate hasn&apos;t processed the payment yet.</shortDescription>
+        <shortDescription>Refund error message when Elevate hasn&apos;t processed the payment yet.</shortDescription>
         <value>Payment can&apos;t be refunded because Elevate hasn&apos;t processed the payment yet.</value>
     </labels>
+    <labels>
+        <fullName>pmtOriginalPaymentNotPaid</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Refund error message when Elevate hasn&apos;t processed the payment yet.</shortDescription>
+        <value>Payment can&apos;t be refunded because the it&apos;s not paid</value>
```

Comment:
Typo here ("because the it's"), but we can cover that when we review the Labels!